generator/stdlib/src/commonMain/kotlin/com/github/jcornaz/kwik/generator/stdlib/Strings.kt

```diff
@@ -45,27 +55,58 @@ fun Generator.Companion.strings(
  * @param charset Set of character to be used in generated strings
  * @param exclude Characters to exclude from generated strings
  */
+@Deprecated(
+    "Use `nonEmptyStrings(maxLength, charGenerator)` instead",
+    ReplaceWith("nonEmptyStrings(maxLength, charGenerator)")
+)
```

Comment:
I think I'd like to keep this overload. At least for now.

```suggestion
@Deprecated(
    "Use `nonEmptyStrings(maxLength, charGenerator)` instead",
    ReplaceWith("nonEmptyStrings(maxLength, charGenerator)")
)
```